WinImageBuilder.psm1

```diff
@@ -930,6 +1031,8 @@ function New-WindowsCloudImage {
             $configValues = @{
                 "InstallUpdates"=$InstallUpdates;
                 "PersistDriverInstall"=$PersistDriverInstall;
+                "PurgeUpdates"=$PurgeUpdates;
+                "DisableSwap"=$DisableSwap;
             }
 
             Generate-UnattendXml $UnattendXmlPath $unattedXmlPath $image $ProductKey $AdministratorPassword

```

Comment:
If there is no product key, this fail. Use something like this:

$params= @{"key"="value"}, then

Set-EntityName  @params